test/docker-compose.yaml

```diff
@@ -37,11 +37,25 @@ services:
   destination_ubuntu:
     build:
       context: ./test
-      dockerfile: dockerfiles/Dockerfile.ubuntu
+      dockerfile: dockerfiles/Dockerfile.debian
+      args:
+        image_tag: ubuntu:22.10
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup
       - ..:/work
     privileged: true
     depends_on: [server]
-    ports:
-      - "127.0.0.1:8220:22"
+    ports: ["127.0.0.1:8220:22"]
+
+  destination_debian:
+    build:
+      context: ./test
+      dockerfile: dockerfiles/Dockerfile.debian
+      args:
+        image_tag: debian:11-slim
+    volumes:
+      - ..:/work
```

Comment:
For some reason using the `/sys/fs/cgroup:/sys/fs/cgroup` volume on debian was causing systemd to error. I don't think I ever found the underlying error, but it was manifesting as no dbus socket, so `systemctl` could not communicate with systemd.

So I've omitted the volume here